google/cloud/vision/v1/vision_gapic.yaml

```diff
@@ -0,0 +1,46 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
+language_settings:
+  java:
+    package_name: com.google.cloud.vision.v1
+  python:
+    package_name: google.cloud.vision.v1
+  go:
+    package_name: google.golang.org/google/cloud/vision/v1
+  csharp:
+    package_name: Google.Cloud.Vision.V1
+  ruby:
+    package_name: Google::Cloud::Vision::V1
+  php:
+    package_name: Google\Cloud\Vision\V1
+interfaces:
+- name: google.cloud.vision.v1.ImageAnnotator
+  collections: []
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 60000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 60000
+    total_timeout_millis: 600000
+  methods:
+  - name: BatchAnnotateImages
+    flattening:
+      groups:
+      - parameters:
+        - requests
+    required_fields:
+    - requests
+    request_object_method: false
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    timeout_millis: 30000
```

Comment:
Fixed manually. The issue for config generator seems filed as https://github.com/googleapis/toolkit/issues/263